ocs_ci/utility/utils.py

```diff
@@ -601,14 +616,16 @@ def exec_cmd(
     masked_cmd = mask_secrets(cmd, secrets)
     log.info(f"Executing command: {masked_cmd}")
     if isinstance(cmd, str):
-        cmd = shlex.split(cmd)
+        # use of shlex has been temporarily removed to fix issue #6489
+        # shlex.split() creates peaks in RSS+VSZ mem usage when called intensively
+        cmd = cmd.split()
```

Comment:
Shlex and split provides different outputs which will cause the regression in our code if we will change it here. It's very dangerous change here.